### PR TITLE
Customisation examples of dockerfile with application-plugin

### PR DESCRIPTION
**Pull request by lanwen**
Relates #389

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -441,6 +441,41 @@ docker {
...
+    instruction {'RUN ls -la'}
+}
+
+docker {
+    javaApplication {
+    }
+}
+----
+
+will result to
```
Comment (cdancy) [inline README.asciidoc]: Lets change to `will result in`

File: README.asciidoc [hunk 2]
```diff
@@ -441,6 +441,41 @@ docker {
 }
 ----
 
+[TIP]
```
Comment (cdancy) [inline README.asciidoc]: How this is formatted is kind of odd IMO. Can we just do a typical "section" like we have elsewhere? I think that would suffice.

**Comment by lanwen** [comment]
updated

**Comment by lanwen** [comment]
Any news on merging? :)

**Comment by cdancy** [comment]
My apologies for letting this slip under the radar. Merging now. Thanks @lanwen !!!

**Pull request merged**